assets/scss/templates/_integrations.scss

```diff
@@ -0,0 +1,83 @@
+body {
+  &[data-active-menu="13"] {
```

Comment:
whats means "13" ?

Reply:
Menu item position. As I've said in the PR description, this is not something that is related to the task, but an improvement on how we are handling views in the library. 👌

Reply:
Mmm, esto no puede afectar a la webapp? me podrás poner una captura de que efecto tiene así entiendo mejor?

Reply:
Viendo más abajo entendí. Tal vez hay que ponerle otro nombre para que quede claro que _data-active-menu_ es una clase de uso interno de la librería.

Reply:
Finalmente creo que el nombre de la clase _data-active-menu_ está bien, porque podría llegar a usarse en otros clientes de la librería.

Pero en lugar de usar este selecto (`&[data-active-menu="13"]`), creo que deberíamos usar una clase auxiliar (solo para la librería) en esa página.

Reply:
Es muy buena la observación, @andresmoschini . Este cambio afecta sólo el uso interno de desarrollo de la librería (el menú interno para pasar entre cada página) y actualmente estamos incluyendo esos estilos dentro del build que sale a producción, sumando estilos que no son usados en los proyectos.
Una cosa que se me ocurre es modificar la tarea de `build` para excluír archivos que no sean directamente relacionados a estilos que se necesiten en los proyectos (los tiles del styleguide, los del menú, el layout de la presentación interna de las páginas, este cambio del menú, etc) a modo de improvement. ¿Qué te parece?

Reply:
Con @andresmoschini convenimos en hacer una serie de improvements de cara al futuro. Algunos cambios chicos van a estar incluídos en este PR, mientras que los cambios más profundos van a estar incluídos en otros PRs en tanto y en cuanto exista el tiempo y el espacio para hacerlos.